site/docs/v1/tech/reactor.adoc

```diff
@@ -27,8 +27,7 @@ Then in your FusionAuth instance navigate to [breadcrumb]#Reactor# and enter you
 _Note: You may enter your License Id on multiple instances.
 However, since billing is based upon the total number of monthly active users across all licensed instances it is recommended that you only license those instances you need support for or require Reactor features for._
 
-Immediately after activating, the *Status* may be `Activated, pending connection`. This means that the activation was successful, now FusionAuth is just obtaining a secure connection
-to Reactor. This may take a minute or two, but should change to `Activated, securely connected` after refreshing the page.
+Immediately after activating, FusionAuth must obtain a secure connection to Reactor and may take a minute or two to complete activation. [field]#Licensed# should change to a green checkmark after refreshing the page.
```

Comment:
This might be a good place to add info about the [field]#Optional license text# and why you'd use that. I'd reuse some of the verbiage you added on the reactor API docs.